### PR TITLE
automatically enable --ignore-osdeps under --new-pr and --update-pr

### PR DESCRIPTION
**Pull request by boegel**
fix for #2260

### CODE REVIEW

**Comment by boegel** [comment]
@easybuilders/easybuild-framework-maintainers fix is confirmed in #2260, please review & merge?

File: easybuild/tools/config.py [hunk 1]
```diff
@@ -360,9 +360,10 @@ def init_build_options(build_options=None, cmdline_options=None):
 
         auto_ignore_osdeps_options = [cmdline_options.check_conflicts, cmdline_options.dep_graph,
                                       cmdline_options.dry_run, cmdline_options.dry_run_short,
-                                      cmdline_options.extended_dry_run, cmdline_options.dump_env_script]
+                                      cmdline_options.extended_dry_run, cmdline_options.dump_env_script,
+                                      cmdline_options.new_pr, cmdline_options.update_pr]
```
Comment (damianam) [inline easybuild/tools/config.py]: Doesn't it make sense to add `merge_pr` too?

Comment (boegel) [inline easybuild/tools/config.py]: There's no point, since `--merge-pr` doesn't involve any parsing of easyconfig files.

**Comment by damianam** [comment]
lgtm, going in

**Pull request merged**